remote_templates/ci/.circleci/circleci.tmpl

```diff
@@ -0,0 +1,34 @@
+---
+delimiters
+  - "{{"
+  - "}}"
+---
```

Comment:
Optional frontmatter support. We don't necessarily need it right away but it does add a lot of flexibility.